TAScheduler/AcceptanceTests/test_login.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase, Client
+from TAScheduler.models import User
+
+class SuccessfulLogin(TestCase):
+    def setUp(self):
+        self.client = Client()
+        User.objects.create_user(email_address="user@example.com", password="password")
+
+    def test_login_with_valid_credentials(self):
+        response = self.client.post('/login/', {'email': 'user@example.com', 'password': 'password'})
+        self.assertRedirects(response, '/home/')  # Assuming the user is redirected to home page on successful login
+class InvalidLoginIncorrectPassword(TestCase):
+    def setUp(self):
+        self.client = Client()
+        User.objects.create_user(email_address="user@example.com", password="password")
+
+    def test_login_with_incorrect_password(self):
```

Comment:
maybe test for "" password?

Reply:
How come? I feel like using an actual string just makes it more readable.

Reply:
I mean in addition to your current test not instead of* (sorry for not being clear)

Reply:
good idea.